platform/ios/src/MGLMapView.h

```diff
@@ -1254,6 +1254,11 @@ MGL_EXPORT IB_DESIGNABLE
  `-[MGLVectorSource featuresInSourceLayersWithIdentifiers:predicate:]` and
  `-[MGLShapeSource featuresMatchingPredicate:]` methods on the relevant sources.
 
+ The returned features may also include annotations that do not conform to
+ `MGLFeature`. These annotations will not be object-equal to those that were
+ originally added to the map. To query the map for annotations, use
```

Comment:
It’s difficult to convey the nuances around object wrapping, but maybe this will help:

> The returned features may also include features corresponding to annotations. These features are not object-equal to the `MGLAnnotation` objects that were originally added to the map.